x-pack/plugins/ingest_manager/public/applications/ingest_manager/components/alpha_messaging.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React, { memo } from 'react';
+import styled from 'styled-components';
+import { EuiText } from '@elastic/eui';
+
+const Message = styled(EuiText)`
+  padding: ${props => props.theme.eui.paddingSizes.m};
+`;
+
+Message.defaultProps = {
```

Comment:
I think you can use `.attrs` on styled component instead https://styled-components.com/docs/api#attrs

Reply:
ah yes, forgot about that / its been awhile since I touched styled components. Thanks for the tip :)

Reply:
any reason of keeping this one? :)

Reply:
oops thanks